services/dashboard/views/model/metrics.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0
+ * See accompanying LICENSE file.
+ */
+'use strict';
+
+angular.module('dashboard.metrics', [])
+
+  .factory('Metrics', [function () {
+    return {
+      meter: function (data) {
+        return {
+          name: _decodeName(data.name),
+          values: {
+            count: parseInt(data.count), // TODO: Integer should a number not a string
+            meanRate: data.meanRate,
+            movingAverage1m: data.m1,
+            movingAverage5m: data.m5,
+            movingAverage15m: data.m15
+          }
+        };
+      },
+
```

Comment:
Add reference here to the data type Histogram. Something like:
//TODO datatype: Histogram. Try and do serde using scalajs See #458